src/backend/joanie/tests/base.py

```diff
@@ -0,0 +1,36 @@
+"""
+Common base API test case
+"""
+from datetime import datetime, timedelta
+
+from django.test import TestCase
+
+from rest_framework_simplejwt.tokens import AccessToken
+
+
+class BaseAPITestCase(TestCase):
+    """Base API test case"""
+
+    @staticmethod
+    def _mock_user_token(username, expires_at=None):
```

Comment:
seems like a bad name as you're not mocking anything. More like "get_user_token"?